client/src/modules/AutoTest/components/Exercise/Exercise.tsx

```diff
@@ -48,7 +48,7 @@ function Exercise({ githubId, courseId, courseTask, finishTask }: ExerciseProps)
         <Form form={form} layout="vertical" requiredMark={false} onFinish={submit} onChange={change}>
           {getExercise()}
           <Row justify="center">
-            <Button loading={loading} type="primary" htmlType="submit">
+            <Button loading={loading} type="primary" htmlType="submit" disabled={loading}>
```

Comment:
this does not fix the problem

Reply:
I just retested that, and this fixes at least the rapid clicking issue
[video](https://pomf2.lain.la/f/prdctm1m.mp4)
maybe I'm missing something
I will look into it more if needed